examples/mapbox-layer.js

```diff
@@ -100,5 +63,26 @@ const mbLayer = new Layer({
   }
 });
 
-map.addLayer(mbLayer);
-map.addLayer(vectorLayer);
+const style = new Style({
+  stroke: new Stroke({
+    color: '#319FD3',
+    width: 2
+  })
+});
+
+const vectorLayer = new VectorLayer({
+  source: new VectorSource({
+    url: 'data/geojson/countries.geojson',
+    format: new GeoJSON()
+  }),
+  style: style
+});
+
+const map = new Map({
+  target: 'map',
+  view: new View({
+    center: fromLonLat(center),
+    zoom: 4
```

Comment:
I'm pretty sure removing the `extent` constraint will break when the user pans out of the world north/south bounds. Although admittedly this constraint could simply be the default configuration!

Reply:
[I see](https://1296-4723248-gh.circle-artifacts.com/0/examples/mapbox-layer.html).  Yeah, let's make this the default behavior (not to allow panning off the top/bottom edge of the world.

Reply:
I added the magic view extent back in and opened #9544.